openedx/core/djangoapps/content/learning_sequences/api/tests/test_outlines.py

```diff
@@ -1072,6 +1073,32 @@ def test_special_exam_attempt_data_empty_when_disabled(self, mock_get_attempt_st
         assert mock_get_attempt_status_summary.call_count == 0
         assert len(student_details.special_exam_attempts.sequences) == 0
 
+    @override_waffle_flag(EXAMS_IDA, active=True)
+    @patch.dict(settings.FEATURES, {'ENABLE_SPECIAL_EXAMS': True})
+    @patch('openedx.core.djangoapps.content.learning_sequences.api.processors.special_exams.get_attempt_status_summary')
+    def test_special_exam_attempt_data_exams_ida_flag_on(self, mock_get_attempt_status_summary):
+        _, student_details, _ = self.get_details(
+            datetime(2020, 5, 25, tzinfo=timezone.utc)
+        )
+
+        # Ensure that no calls are made to get_attempt_status_summary
+        assert mock_get_attempt_status_summary.call_count == 0
+        
+    @override_waffle_flag(EXAMS_IDA, active=True)
+    @patch.dict(settings.FEATURES, {'ENABLE_SPECIAL_EXAMS': True})
+    def test_special_exam_attempt_data_exam_type(self):
+        _, student_details, _ = self.get_details(
+            datetime(2020, 5, 25, tzinfo=timezone.utc)
+        )
+
+        # Ensure that exam type is correct
+        for sequence_key in self.get_sequence_keys(exclude=[self.seq_normal_key]):
+            assert sequence_key in student_details.special_exam_attempts.sequences
+            attempt_summary = student_details.special_exam_attempts.sequences[sequence_key]
+            assert type(attempt_summary) == dict  # lint-amnesty, pylint: disable=unidiomatic-typecheck
+            assert attempt_summary["suggested_icon"] == "fa-pencil-square-o"
+            assert "Exam" in attempt_summary["short_description"]
```

Comment:
I feel like there could be a better way to test that the description matches the exam type, but not sure how to do this in a way that's not too cumbersome. Wondering how to maybe better utilize the ExamData data type.